modules/lang-painless/src/test/resources/rest-api-spec/test/painless/70_execute_painless_scripts.yml

```diff
@@ -23,3 +38,53 @@
           context:
             painless_test: {}
   - match: { result: "-90" }
+
+---
+"Execute with filter context":
```

Comment:
Is there any way these could be pure unit tests? At least single node tests?

Reply:
I will look into this.